golem-base-indexer/golem-base-indexer-migration/src/m20250923_081124_create_entity_data_size_histogram_matview.rs

```diff
@@ -0,0 +1,154 @@
+use sea_orm_migration::{
+    prelude::*,
+    sea_orm::{DatabaseBackend, Statement, TransactionTrait},
+};
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        let create_view = Statement::from_string(
+            DatabaseBackend::Postgres,
+            r#"
+CREATE MATERIALIZED VIEW golem_base_entity_data_size_histogram AS
+WITH entities AS (
+    SELECT 
+        OCTET_LENGTH(data) as size
+    FROM golem_base_entities
+    WHERE 
+        status = 'active' 
+        AND data IS NOT NULL 
+        AND OCTET_LENGTH(data) > 0
```

Comment:
I think it would make sense to show active entities with empty data on the chart.